MatrixSDK/Data/MXRoom.m

```diff
@@ -3643,7 +3643,7 @@ - (void)validateEncryptionStateConsistency
     if (!crypto)
     {
 #ifdef MX_CRYPTO
-        MXLogFailure(@"[MXRoom] checkEncryptionState: Crypto module is not present");
+        MXLogError(@"[MXRoom] checkEncryptionState: Crypto module is not present");
```

Comment:
There are some tests where these conditions actually occur. I do not yet know whether this is expected or not, either way the assertion will stop the tests. Temporarily switching to logging errors instead of failures and will investigate the failing tests individually.